src/main/java/com/twilio/oai/resolver/csharp/CsharpCodegenModelDataTypeResolver.java

```diff
@@ -73,9 +73,12 @@ private void resolveEnum(CodegenProperty property) {
         }
         String className = OperationStore.getInstance().getClassName();
         property.isEnum = true;
-        property.enumName = property.complexType.contains("Enum") || property.complexType.contains("enum")
-                ? Utility.removeEnumName(property.complexType) + ApplicationConstants.ENUM 
+        if(property.complexType != null){
+            property.enumName = property.complexType.contains("Enum") || property.complexType.contains("enum")
+                ? Utility.removeEnumName(property.complexType) + ApplicationConstants.ENUM
```

Comment:
Check if it is not breaking existing code.
Also check if an Enum defined as referenced is used at multiple places.
For Example:
Enum is defined under component.schema
component.schema.UpdateStatusEnum

And above enum is referenced at 2 or more places.